nautobot/dcim/tables/devices.py

```diff
@@ -173,6 +173,7 @@ class DeviceTable(StatusTableMixin, RoleTableMixin, BaseTable):
         template_code="""{% if record.device_redundancy_group %}<span class="badge badge-default">{{ record.device_redundancy_group_priority|default:'None' }}</span>{% else %}—{% endif %}"""
     )
     controller_managed_device_group = tables.Column(linkify=True)
+    software_version = tables.Column(linkify=True)
```

Comment:
@HanlinMiao You might want to add verbose_name so it looks correctly capitalized.